daemon/src/types.rs

```diff
@@ -139,8 +139,8 @@ pub enum JSONRPCFromEditor {
 }
 impl JSONRPCFromEditor {
     pub fn from_jsonrpc(jsonrpc: &str) -> Result<Self, anyhow::Error> {
-        let error_message = format!("Failed to deserialize editor message: {jsonrpc}");
-        let message = serde_json::from_str(jsonrpc).expect(&error_message);
+        //let error_message = format!("Failed to deserialize editor message: {jsonrpc}");
+        let message = serde_json::from_str(jsonrpc)?; //.context(error_message)?;
```

Comment:
Should we remove the comments here, aka, are you happy with the "question mark only"? Or was this something not yet working?

Reply:
Ah no, it was an oversight. I removed the comments.